Source/ChocolateyGui/Services/ServiceCallbackHandler.cs

```diff
@@ -15,27 +15,27 @@ public ServiceCallbackHandler(IProgressService progressService)
             _progressService = progressService;
         }
 
-        public void LogMessage(StreamingLogMessage message)
+        public void LogMessage(LogMessage message)

```

Comment:
Ahh, never mind.

Reply:
☝️

Reply:
It's likely the service callback handler could go away?